docs/Installation.md

```diff
@@ -6,13 +6,14 @@ Welcome to the Kunstmaan Bundles Standard Edition - a fully-functional CMS (cont
 
 This document contains information on how to download, install, and start using the Kunstmaan Bundles CMS.
 
-We assume you have read the official Symfony documentation and know the basics of creating Symfony applications.
+This guide assumes that you have read the [official Symfony documentation][2] and know the basics of creating
+Symfony applications.
 
 
-1) Minimal requirements
+1) Minimal Requirements
 -----------------------
 
-Since we use the Symfony framework, the [minimal requirements of Symfony][2] apply, so :
+Since we use the Symfony framework, the [minimal requirements of Symfony][3] apply, so:
```

Comment:
the trouble of using numbered link references. I would recommend to use an unique string instead